language/stdlib/modules/LibraAccount.move

```diff
@@ -1110,6 +1115,41 @@ module LibraAccount {
             chain_id,
         )
     }
+    spec fun module_prologue {
+        pragma verify;
+        let transaction_sender = Signer::spec_address_of(sender);
+        let max_transaction_fee = txn_gas_price * txn_max_gas_units;
+        include AbortsIfModulePrologue<Token> {
+            sender,
```

Comment:
You don't need to pass this. A name in the context is automatically matched. You would get an error if not.

I'm actually surprised this even works (only name: value pairs should work). I guess it is just ignored where an error should have been produced if you use this notation.

Reply:
ah! awesome. I'll update :)

Reply:
Updated, but it seems like the name context doesn't take into account local (`let`) bindings. Not sure if this is meant to be the case or not?